td-loader/fuzz/fuzz_targets/pe.rs

```diff
@@ -0,0 +1,27 @@
+#![no_main]
```

Comment:
What's the relationship between td-shim/fuzz/fuzz_pe_loader/main.rs and this file?

Reply:
td-shim/fuzz/fuzz_pe_loader/main.rs It is afl, now libfuzzer is added, the fuzzing command is different but the fuzzing data is the same.